vars/helmPullChart.groovy

```diff
@@ -1,7 +1,8 @@
-void call(String chart, Map opts = [:]) {
+void call(String chart, Map opts = [:], String helmChartVersion = '') {
   Map containers = opts.containers ?: [:]
   String helmContainer = containers.helm ?: ''
   List<String> helmOpts = opts.helmOpts ?: ["--entrypoint=''"]
+  
 
   println "[ace] - Got containers - ${containers}"
 
```

Comment:
Not able to make a comment on hte line below for some reason.. 
In the definition of aceContainerWrapper it takes an option helmOpts. This is never used.. What was/is the intended way this should work, @ekarlso ?

Reply:
:pray: can @bjorges or @Starefossen explain how the opts are connected to the ace.yaml file? Opts are defined in the jekinsfile in the source code repository. 

:man_shrugging: Is there any specification on how to use the opts combined with ace.yaml file? And how about the `opts.helmOpts` :eyes:. I hope we can get the chart and chart version defined in the ace.yaml file and use this to pull the chart with correct version from the chart repo.